runners/apex/src/main/java/org/apache/beam/runners/apex/translators/functions/ApexParDoOperator.java

```diff
@@ -104,14 +102,16 @@ public ApexParDoOperator(
       List<TupleTag<?>> sideOutputTags,
       WindowingStrategy<?, ?> windowingStrategy,
       List<PCollectionView<?>> sideInputs,
-      Coder<WindowedValue<InputT>> inputCoder
+      Coder<WindowedValue<InputT>> inputCoder,
+      StateInternalsFactory<Void> stateInternalsFactory
```

Comment:
This is called during translation only. The operator state is the written to an initial checkpoint, from which it is loaded when deployed into the execution environment.